common/script/mapping-tables.js

```diff
@@ -175,9 +157,9 @@ function mappingTables() {
 
       // create content for each <details> element; add row header's content to summary
       var details = document.createElement('details');
-      details.className = 'map removeOnSave';
+      details.className = 'map';
```

Comment:
This change makes it such that if you load the html page, click the "respec" menu button, then "export", the exported document has the "summary view" instead of the tables. Is this what we want?